docs/user/dashboard/dashboard.asciidoc

```diff
@@ -0,0 +1,472 @@
+[[dashboard]]
+= Dashboard
+
+[partintro]
+--
+
+A _dashboard_ is a collection of panels that provide you with an overview of your data that you can 
+present and share. On a dashboard, you can add a variety of panels that you can rearrange to build a 
+dashboard that tells a story about your data. Panels contain everythng you need, including visualizations, 
```

Comment:
Multiple usages of 'dashboard' in one sentence strike me as odd - could be changed to something like "On a dashboard, you can add a variety of panels that you can rearrange to tell a story about your data"

Also typo at - 'everyth**i**ng you need'